common/src/misc/webauthn_iframe.ts

```diff
@@ -20,6 +20,7 @@ export class WebAuthnIFrame extends IFrameComponent {
             this.platformUtilsService.launchUri(`${this.webVaultUrl}/webauthn-fallback-connector.html?${params}`);
         } else {
             super.initComponent(params);
+            this.iframe.allow = 'publickey-credentials-get ' + new URL(this.webVaultUrl).origin;
```

Comment:
Did we test this? I'm curious if we can set the `src` prior to setting `allow`?

Reply:
It worked for me, yeah. I was kind of surprised by that too, but I just re-tested in an incognito window so I don't think it's a caching false-positive